src-json/define.json

```diff
@@ -122,20 +122,29 @@
 		"doc": "Erase generic classes on C#",
 		"platforms": ["cs"]
 	},
+	{
+		"name": "EvalCallStackDepth",
```

Comment:
One thing I've wondered, isn't having both `name` and `define` a bit redundant (i.e. couldn't one be derived from the other, or are there exceptions)?

Reply:
Originally you could probably derive the defines from the OCaml name (except the unfortunate case of `HxcppSmartStings`, which somehow went by unnoticed). Since dot notation was introduced into metadata, the mapping may be less obvious – there are still many fields which use the un-dotted identifier. For defines this might be easier?

But to be honest I did not thoroughly think about unifying `name` and `define` (or `metadata`). I also think it's fine if it is done as part of another PR at some point.